Solutions/CohesitySecurity/json_parser.sh

```diff
@@ -6,7 +6,7 @@ cd "$SCRIPTPATH"
 ClientId=$(cat ./cohesity.json | jq '."ClientId"' | sed 's/^"//g;s/"$//g')
```

Comment:
In the next PR, please update the other variables in cohesity.json and json_parser.sh to follow the snake case like 'api_key'.